db/migrate/20180710171515_create_targets.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTargets < ActiveRecord::Migration[5.1]
+  def change
+    create_table :targets do |t|
+      t.string :title
+      t.decimal :lat
+      t.decimal :lng
```

Comment:
Try to avoid this kind of abbreviatons (lat and lng)

Reply:
Also you can add database not null validations for the ones that you are validating the presence in the model